doc/programming/language/types.rst

```diff
@@ -439,6 +439,25 @@ set. Internally, Spicy stores them as UTF-8.
 
 .. include:: /autogen/types/string.rst
 
+.. _type_struct:
+
+Struct
+------
+
```

Comment:
Please add at least a couple of sentences of an introductory summary here (see the other types for examples).

Reply:
I'll add

> A structs is a heterogeneous container of a fixed, ordered set of named values similar to a Tuple.